test/rules.js

```diff
@@ -832,6 +832,21 @@ const tests = {
                 errors: ['sotd.new-features.no-link'],
             },
         ],
+        usage: [
+            {
+                doc: 'p2021/2021-ry.html',
+                config: {
+                    longStatus: 'Registry',
+                },
+            },
+            {
+                doc: 'p2021/2021-rec.html',
+                config: {
+                    longStatus: 'Registry',
+                },
+                errors: ['sotd.usage.not-found'],
```

Comment:
Same question as above. What are you trying to test? Obviously, using a REC template will not work for Registry

Reply:
... to test the wording doesn't match and pubrules should reports an error